web/index.html

```diff
@@ -461,71 +526,52 @@ <h3>Used in</h3>
                     let query = this.searchQuery.toLowerCase()
                     return this.data.list.filter(x => x[1].toLowerCase().includes(query)).map(x => x[0]).slice(0, this.loaded)
                 },
-                craftingTree: function () {
```

Comment:
craftingTree can no longer be computed as when save data is loaded it will need to be able to await chunks loading which does not work for computed functions